ws-server/src/future.rs

```diff
@@ -92,6 +101,12 @@ where
 			}
 		}
 	}
+
+	fn poll_heartbeat(&mut self, cx: &mut Context) {
```

Comment:
```suggestion
	fn poll_stop_monitor(&mut self, cx: &mut Context) {
```

(Not an accurate name I know, but gives a hint in the right direction?)